onnxruntime/test/contrib_ops/element_wise_ops_test.cc

```diff
@@ -219,13 +218,15 @@ TEST(MathOpTest, ComplexMul) {
   tester.AddOutput<float>("C", {4, 2}, output_data);
 
   std::vector<std::unique_ptr<IExecutionProvider>> execution_providers;
+#ifdef USE_CUDA
   execution_providers.push_back(DefaultCudaExecutionProvider());
+#elif USE_ROCM
```

Comment:
should be consistent here, either `#if #elif` or `#if defined   #elif defined`

Reply:
Fixed in 5bc0f95393d6f076ee7cf178d32ceab8e856a91d.